recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -188,7 +195,7 @@ forward)
     pcr_params="${pcr_params} -p 15:${pcr15}"
 
     # Calculate DRTM PCRs only when booting legacy (ie. PCR8 is all 0)
-    if [[ "${pcr8}" = *[0]* ]]; then
```

Comment:
This indeed is buggy, should have been `if [[ ! "${pcr8}" = *[!0]* ]]; then`

Reply:
If there is no strong feeling for one ( `-z "${var//0}"`) or the other (`[[ ! "${var}" = *[!0]* ]]`) and since this PR is out both in master and stable-8, I think this is good to be merged soon?

Reply:
TBH would be more useful to introduce a function that checks if a string is all a given character, not just 0, so that it could be used to check if drtm PCRs are set (which are all `f`s by default).